src/graph.cc

```diff
@@ -507,9 +507,9 @@ bool ImplicitDepLoader::LoadDeps(Edge* edge, string* err) {
   return true;
 }
 
-bool ImplicitDepLoader::LoadDepFile(Edge* edge, const string& path,
-                                    string* err) {
-  METRIC_RECORD("depfile load");
+bool ImplicitDepLoader::LoadDepfileDeps(Edge* edge, string path,
+                                        vector<Node*>* depfile_deps,
```

Comment:
Consider using std:: for types from the standard library for newly added code.

Reply:
Thanks for pointing out the upcoming rule change!